catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/operations/ResourceOperationsTest.java

```diff
@@ -0,0 +1,500 @@
+        when(queryResponseMock.getResults()).thenReturn(mockResultList);
+        when(queryResponseMock.getResults()
+                .get(0)
+                .getMetacard()).thenReturn(metacard);
+        
+        when(reliableResourceDownloadManagerMock.download(any(ResourceRequest.class), any(MetacardImpl.class), any(ResourceRetriever.class))).thenReturn(
+                resourceResponseMock);
+
+        Resource resourceMock = mock(Resource.class);
+        when(resourceResponseMock.getResource()).thenReturn(resourceMock);
+
+ 
+        frameworkProperties.setReliableResourceDownloadManager(reliableResourceDownloadManagerMock);
+
+        resourceOperations.getResource(resourceRequestMock,
+                isEnterprise,
+                resourceName,
+                fanoutEnabled);
+
+    }
+
+    @Test
+    public void testValidateFixGetResourceReturnsResourceResponse()
+            throws ResourceNotFoundException, IOException, ResourceNotSupportedException,
+            FederationException, UnsupportedQueryException, DownloadException {
+
+        boolean isEnterprise = true;
+        boolean fanoutEnabled = true;
+        String resourceName = "bob";
+
+        Result resultMock = mock(Result.class);
+
+        ArrayList<Result> mockResultList = new ArrayList<>();
+        mockResultList.add(resultMock);
+
+        when(resourceRequestMock.getAttributeValue()).thenReturn(testUri);
+        when(resourceRequestMock.getAttributeName()).thenReturn("resource-uri");
+        when(queryOperationsMock.query(any(QueryRequest.class), any(FederationStrategy.class), anyBoolean(), anyBoolean())).thenReturn(
+                queryResponseMock);
+        when(queryResponseMock.getResults()).thenReturn(mockResultList);
+        when(queryResponseMock.getResults()
+                .get(0)
+                .getMetacard()).thenReturn(metacard);
+
+        when(reliableResourceDownloadManagerMock.download(any(ResourceRequest.class), any(MetacardImpl.class), any(ResourceRetriever.class))).thenReturn(
+                resourceResponseMock);
+
+        Resource resourceMock = mock(Resource.class);
+        when(resourceResponseMock.getResource()).thenReturn(resourceMock);
+
+ 
+        frameworkProperties.setReliableResourceDownloadManager(reliableResourceDownloadManagerMock);
+
+        resourceOperations.getResource(resourceRequestMock,
+                isEnterprise,
+                resourceName,
+                fanoutEnabled);
+
+    }
+
+    @Test(expected = ResourceNotFoundException.class)
+    public void testValidateFixGetResourceReturnsResourceResponseThrowsResourceNotFoundException()
+            throws ResourceNotFoundException, IOException, ResourceNotSupportedException,
+            FederationException, UnsupportedQueryException, DownloadException {
+
+        boolean isEnterprise = true;
+        boolean fanoutEnabled = true;
+        String resourceName = "bob";
+
+        Result resultMock = mock(Result.class);
```

Comment:
Looks like 245-260 and 263 is duplicated between multiple tests. I think you could pull this out into a helper method.